README.md

```diff
@@ -26,28 +81,45 @@ services:
     volumes: 
       # Where audio files from .sound commands should be placed
       - ./audio:/app/audio
+      # Data folder
+      - ./data:/app/data
 
 ```
 
-To run it simply execute docker compose command, depending on compose engine installed on your system.
+## Debug and development <a id="tc2"></a>
 
-```bash
-# V1
-docker-compose up -d
-# V2
-docker compose up -d
-```
+Here are stored important informations for debugging and development of bot.
 
-### Using docker command
+### Database schemas <a id="tc21"></a>
 
-Pulling and running container could be done using command below, simply place the TOKEN and name, and set the volume path in PWD.
+Here are table schemase currently implemented. Detailed short names:
 
-```bash
-docker run --rm -d -e "TOKEN=<discord app client secret here>" --name <name> -v <PWD>/audio:/app/audio seker212/discord_bot:2.0
-```
+- PK -> Primary key
+- FK -> Foreign key
+- NN -> Not null
+- AI -> Auto increment
+
+`Config` table
+
+| Name | Type | PK | FK | NN | AI |
+|------|------|------|------|------|------|
+| Id | Integer | X |  | X | X |
+| GuildId | Text |   |  |   |   |
+| ParameterName | Text |   |  |   |   |
```

Comment:
Isn't ParameterName NN?

Reply:
It is not in the database file schema. In code yes, but schema does not define it like this.